Mlem/Models/Trackers/SiteInformationTracker.swift

```diff
@@ -39,8 +43,14 @@ class SiteInformationTracker: ObservableObject {
                 myUserInfo = response.myUser
                 allLanguages = response.allLanguages
                 if let userInfo = response.myUser {
-                    myUser = UserModel(from: userInfo.localUserView.person)
+                    // need to fetch full user info to get moderated communities, required to display mod functions from non-community feed
+                    myUser = try await personRepository.loadUser(for: userInfo.localUserView.localUser.personId)
                     myUser?.isAdmin = response.admins.contains { $0.person.id == myUser?.userId }
+                    if let communities = myUser?.moderatedCommunities {
+                        moderatedCommunities = Set(communities.map(\.communityId))
+                    } else {
+                        moderatedCommunities = .init(minimumCapacity: 1)
+                    }
```

Comment:
You can get the list of moderated communities from MyUserInfo without fetching the ApiPersonResponse separately